projects/angular/src/utils/variables/API.md

```diff
@@ -0,0 +1,79 @@
+| --clr-h2-font-weight        | Changes the font thickness of the main heading and h2 heading.
+| --clr-h2-font-family        | Changes the text's font style of the main heading and h2 heading.
+| --clr-h3-color              | Changes the color of the text in the h3 heading and h3 content.
+| --clr-h3-font-weight        | Changes the font thickness of the and h3 heading.
+|--clr-h3-font-family         | Changes the text's font style of h3 heading.
+| --clr-h4-color              | Changes the color of the text in the h4 heading and h4 content.
+| --clr-h4-font-weight        | Changes the font thickness of the and h4 heading.
+| --clr-h4-font-family        | Changes a text's font style of h4 heading.
+| --clr-h5-color              | Changes the color of the text in the h5 heading and h5 content.
+| --clr-h5-font-weight        | Changes the font thickness of the and h5 heading.
+| --clr-h5-font-family        | Changes a text's font style of h5 heading.
+| --clr-h6-color              | Changes the color of the text in the h6 heading and h6 content.
+| --clr-h6-font-weight        | Changes the font thickness of the and h6 heading.
+| --clr-h6-font-family        | Changes a text's font style of h6 heading.
+| --clr-p0-color              | Changes the text color of content in p0 .
+| --clr-p0-font-weight        | Changes the text thickness of content in p0.
+| --clr-p1-color              | Changes the text color of content in p1 .
+| --clr-p1-font-weight        | Changes the text thickness of content in p1.
+| --clr-p2-color              | Changes the text color of content in p2 .
+| --clr-p2-font-weight        | Changes the text thickness of content in p2 .
+| --clr-p3-color              | Changes the text color of content in p3 .
+| --clr-p3-font-weight        | Changes the text thickness of content in p3 .
+| --clr-p4-color              | Changes the text color of content in p4 .
+| --clr-p4-font-weight        | Changes the text thickness of content in p4 .
+| --clr-p5-color              | Changes the text color of content in p5.
+| --clr-p5-font-weight        | Changes the text thickness of content in p5 .
+| --clr-p6-color              | Changes the text color of content in p6 .
+| --clr-p6-font-weight        | Changes the text thickness of content in p6 .
+| --clr-p7-color              | Changes the text color of content in p7 .
+| --clr-p7-font-weight        | Changes the text thickness of content in p7 .
+| --clr-p8-color              | Changes the text color of content in p8 .
+| --clr-p8-font-weight        | Changes the text thickness of content in p8 in .
+
+## class
+
+| Class name                   | Description                              |
+| ---------------------------- | ---------------------------------------- |
+| clr-typography-demo          | This is a container class| 
+| clr-typography-font-weight   | Used to set how thick or thin characters in text should be displayed |
```

Comment:
Neither of these are defined in the CSS. I search for `.clr-typography-demo` and `.clr-typography-font-weight` and got zero hits for both.

Reply:
Removed, As suggested